frontend/src/metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.styled.tsx

```diff
@@ -131,7 +131,7 @@ export const NativeQueryEditorRoot = styled.div`
   width: 100%;
   background-color: var(--mb-color-bg-light);
 
-  ${aceEditorStyle}
+  ${() => getAceEditorStyles()}
```

Comment:
Looks like we have `color("text-dark")` in getAceEditorStyles - let's address this in the next PR when addressing `text-*` and friends